x962/curves.json

```diff
@@ -697,6 +707,7 @@
       },
       "order": "0x155555555555555555555555610c0b196812bfb6288a3ea3",
       "cofactor": "0x6",
+      "seed": "0xE053512DC684D696E676875615175067AE786D1F",
```

Comment:
The seed is not formatted the same way, in some cases it is `0xhex` in others its just `hex`. I would prefer just `hex`.

Reply:
Fixed.